### PR TITLE
Add Instagram to Social

### PR DESCRIPTION
**Pull request by dsoegijono**

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -44,6 +44,12 @@ websites:
       software: Yes
       doc: https://help.hootsuite.com/entries/22527304-Managing-Google-Authenticator
 
+    - name: Instagram
+      url: htpps://www.instagram.com
```
Comment (mxxcon) [inline _data/social.yml]: it's https, not htpps :smile:

**Comment by jdavis** [comment]
If you think Instagram should have 2FA, argue that case with taking the [Site Criteria](https://github.com/jdavis/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria) into consideration.

**Comment by mxxcon** [comment]
Instagram has the same impact as Twitter..

**Comment by dsoegijono** [comment]
@jdavis I agree with @mxxcon that Instagram has the same impact as Twitter. I think it fits with criteria 1 for Personal Image.

**Comment by jdavis** [comment]
Okay, cool. If you two think it should be on there, then it shall be!

**Pull request merged**